src/containers/entitysetsearch/AdvancedDataSearch.js

```diff
@@ -68,13 +68,15 @@ export default class AdvancedDataSearch extends React.Component {
     });
     AuthorizationApi.checkAuthorizations(accessChecks)
     .then((response) => {
-      const propsWithReadAccess = [];
+      const propsWithReadAccess = new Set();
```

Comment:
If you care about order, you can check out `OrderedSet` from `immutable.js`:

https://facebook.github.io/immutable-js/docs/#/OrderedSet